tests/components/conftest.py

```diff
@@ -118,3 +120,11 @@ def mock_light_entities() -> list["MockLight"]:
         MockLight("Ceiling", STATE_OFF),
         MockLight(None, STATE_OFF),
     ]
+
+
+@pytest.fixture
+def mock_switch_entities() -> list[MockToggleEntity]:
```

Comment:
A ToggleEntity isn't the same as a SwitchEntity.

Reply:
The old test relayed also on `MockToggleEntity` before, I just kept that (https://github.com/home-assistant/core/pull/114305/files#diff-0c5c0cf8f3c575b49727e26975779a7a072bc748148e3b5bd45922de8bbbfae2L21-L23)

Reply:
Right, but now we're saying that these are switch entities, which is not true.

Reply:
Right, renamed this to mock_toggle_entities.